### PR TITLE
[New] MST VPC peering

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

Added:
VPC peering for AWS, GCP, Azure
Attaching VPC transit gateway to AWS.

# Links

https://app.intercom.com/a/apps/uvcadzno/articles/articles/5452806/show - AWS transit gateway

Looks like this can be done using the Aiven Console now. 

https://app.intercom.com/a/apps/uvcadzno/articles/articles/5489252/show - VPC peering - Azure

Created the VPC peering for GCP using the Aiven docs.

Fixes https://github.com/timescale/docs/issues/2106

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

Please review the content for technical accuracy and completeness. I have not validated the steps. I have only used the resources available for these from Aiven docs and the KB.

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [ ] Is the content free from typos?
*   [ ] Does the content use plain English?
*   [ ] Does the content contain clear sections for concepts, tasks, and references?
*   [ ] Have any images been uploaded to the correct location, and are resolvable?
*   [ ] If the page index was updated, are redirects required
      and have they been implemented?
*   [ ] Have you checked the built version of this content?

### CODE REVIEW

**Comment by spaceshipoperator** [comment]
hello @Rajakavitha1 thank you for including me as reviewer; I got a request from a customer just today and I think they may benefit from this information; great timing!  However, I cannot seem to access the intercom.com link (I tried with my timescale gmail; are there otherwise instructions I need to follow to setup/access?)

I will read the md and look for any obvious issues; eager to approve so I can share with Sergey at ravenyou.io

File: mst/vpc-peering/vpc-peering-azure.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
+---
+title: Set up Virtual Private Cloud (VPC) peering on Azure
+excerpt: Secure your Managed Service for TimescaleDB instance with VPC peering on Azure
+products: [mst]
+keywords: [vpc, peer, azure]
+---
+
+# Configure VPC peering on AWS
```
Comment (spaceshipoperator) [inline mst/vpc-peering/vpc-peering-azure.md:8]: should say Azure

**Review by spaceshipoperator — CHANGES_REQUESTED** [review]
I am not directly familiar with setting up peering with Azure; seems many more steps than AWS; I've done AWS, not GCP but it looks very similar

I found one (minor) typo needs to change, but otherwise I think I have a customer that can benefit from this today and would like to approve

File: mst/vpc-peering/vpc-peering-azure.md (hunk 1 above)
Comment (Rajakavitha1) [reply mst/vpc-peering/vpc-peering-azure.md]: fixed

**Comment by Rajakavitha1** [comment]
> hello @Rajakavitha1 thank you for including me as reviewer; I got a request from a customer just today and I think they may benefit from this information; great timing! However, I cannot seem to access the intercom.com link (I tried with my timescale gmail; are there otherwise instructions I need to follow to setup/access?)
> 
> I will read the md and look for any obvious issues; eager to approve so I can share with Sergey at ravenyou.io

Hi @spaceshipoperator  you may want to view the updated content in the preview http://docs-dev.timescale.com/docs-vpc-rajie. For the access to intercom.com you may want to reach out to @carlotas19 . She helped me with the access to intercom.com

**Comment by Loquacity** [comment]
IKt's building now! 👍 http://docs-dev.timescale.com/docs-vpc-rajie/mst/vpc-rajie/vpc-peering/

**Comment by Rajakavitha1** [comment]
> It's building now! 👍 http://docs-dev.timescale.com/docs-vpc-rajie/mst/vpc-rajie/vpc-peering/

Thank you @Loquacity !!!!!

**Review by spaceshipoperator — APPROVED** [review]

File: mst/vpc-peering/vpc-peering-azure.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+    In case you get this error, it's possible the role assignment hasn't taken
+    effect yet. If that is the case, try logging in again and creating the
+    peering again after waiting a bit by repeating the commands in this step. If
+    the error message persists, check the role assignment was correct.
+
+    The client `<random uuid>` with object id `<another random uuid>` does not have
+    authorization to perform action
+    `Microsoft.Network/virtualNetworks/virtualNetworkPeerings/write` over scope
+    '$user_vnet_id' If access was recently granted, refresh your credentials.
```
Comment (charislam) [inline mst/vpc-peering/vpc-peering-azure.md:246]: Maybe put this inside a code block to make it clearer that it's an error message

File: mst/vpc-peering/vpc-peering-gcp.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+1.  Type the project ID of your GCP project in `GCP Project ID`.
+
+1.  Type the network name of the VPC in GCP in `GCP VPC network name`.
+
+1.  Click `Add peering connection`.
+
+    A new connection with the status as `Pending Peer` is listed in your GCP
+    console. Make a note of the project name and the network name.
+
+1.  In the GCP console go to `VPC` > `VPC network peering` and select
```
Comment (charislam) [inline mst/vpc-peering/vpc-peering-gcp.md:38]:
```suggestion
1.  In the GCP console, go to `VPC` > `VPC network peering` and select
```

**Review by charislam — APPROVED** [review]

File: mst/vpc-peering/vpc-peering-azure.md (hunk 2 above)
Comment (Rajakavitha1) [reply mst/vpc-peering/vpc-peering-azure.md:246]: Thanks @charislam !!! fixed

**Pull request merged**